src/templates/base.html

```diff
@@ -197,7 +197,16 @@ <h4 class="ui inverted header">Footer Header</h4>
         // Websockets
         //WS_SUBMISSION_LISTEN: ""
 
-        // Competitions
+        // Analytics
+        ANALYTICS_API: function (start, end, time_unit, format) {
+            let parameters = '?start_date=' + start + '&end_date=' + end + '&time_unit=' + time_unit
+            if (format != null) {
+                parameters += '&format=' + format
+            }
```

Comment:
should be able to make use of `$.param(filters)` and pass an object to this function, instead of the individual variables
```javascript
filters = {start_date: 'whatever', end_date: 'whatever'}
ANALYTICS_API(filters)
```
```javascript
ANALYTICS_API: function (filters) {
   return `{% url ... %}/?${$.param(filters)}`
}
```
(can't remember if the `/?` is necessary w/ this syntax or not)

Reply:
Done.